prometheus/testutil/testutil.go

```diff
@@ -41,8 +41,10 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"testing"
 
 	"github.com/prometheus/common/expfmt"
+	"github.com/stretchr/testify/require"
```

Comment:
Hm.. is there a way to avoid this package? For a reason we wanted to make sure it's not used, so we are not leaking it.

Since this is not testing code, every project that imports `client_golang` needs to import `testify`....  I think we want to copy the diff function and maintain here.


See how we did that here https://github.com/efficientgo/tools/blob/main/core/pkg/testutil/testutil.go#L101

Reply:
I totally agree. I think I was a little hasty to review this, and I haven't noticed testify is not there already :see_no_evil:

Reply:
I guess because it's already a transitive dependency.

Reply:
yeah, I agree too, this makes sense
but the above diff function has a dep on
https://github.com/efficientgo/tools/blob/fe763185946be83b20da626605319733bd7f97cb/core/pkg/testutil/testutil.go#L16 which is not maintained should I transfer the module implementation over here or just simply import it.